src/Adapter/Validate.php

```diff
@@ -33,6 +33,8 @@
  */
 class Validate
 {
+    public const MYSQL_UNSIGNED_INT_MAX = 4294967295;
```

Comment:
```suggestion
    public const MYSQL_UNSIGNED_INT_MAX = 4294967295; // max 32 bits value: (2^32)-1 
```

Reply:
Just a suggestion so we don't think this number just came out of the blue

Reply:
Fixed

Reply:
I like the technical solution proposed here I'm just not convinced at all with the location of the const (in an Adapter service class) Especially since this namespace is destined to be moved

I would suggest alternatives:
- put the const in the legacy `ValidateCore` class for now (in the absence of a better solution)
- create a dedicated constraint `UnsignedInt` in the `PrestaShop\PrestaShop\Core\ConstraintValidator\Constraints` namespace (but it's more work)
- put this const in a related VO but the issue is that this notion is quite generic and not related to a specific domain value

So for now my best proposal would be to have this in the `ValidateCore`